packages/block-editor/src/components/off-canvas-editor/style.scss

```diff
@@ -1 +1,15 @@
-//Styles for off-canvas editor, remove this line when you add some css to this file!
+.offcanvas-editor-list-view-tree-wrapper {
+	max-width: 100%;
+	overflow-x: auto;
+	.block-editor-list-view-block__menu-cell:last-child {
+		padding-right: 0;
+	}
+
+	.block-editor-list-view-leaf {
+		display: block;
+		max-width: $sidebar-width - (2 * $grid-unit-20);
```

Comment:
What does `2 * $grid-unit-20` represent? Sidebar padding? If so shall we make a Sass var to represent that?

Reply:
yes, it's sidebar padding, I just used the same var that the actual sidebar was using, since it's not defining one itself!

Reply:
Oh that's weird. I just see this and think "oh that represents something but I'm not sure what".

If it was a var `$sidebar-padding` I wouldn't have to interpret the code and it would be more readable.